test/unit/language-extensions/multi.spec.ts

```diff
@@ -0,0 +1,115 @@
+import * as path from "path";
+import * as util from "../../util";
+import * as tstl from "../../../src";
+import {
+    invalidMultiFunctionUse,
+    invalidMultiTypeToNonArrayBindingPattern,
+    invalidMultiTypeArrayBindingPatternElementInitializer,
+} from "../../../src/transformation/utils/diagnostics";
+
+const multiProjectOptions: tstl.CompilerOptions = {
+    types: [path.resolve(__dirname, "../../../language-extensions")],
+};
+
```

Comment:
Could you add one simple test that tests the main use case and also serves as example on how to use this. If I understand correctly it should be something like:
```ts
util.testModule`
    function multiReturn(): MultiReturn<[string, number]> {
        return $multi("foo", 5);
    }

    const [a, b] = multiReturn();
    export { a, b };
`.expectToEqual({a: "foo", b: 5});
```

Reply:
Added this example use case